addon/components/frost-list.js

```diff
@@ -117,9 +117,11 @@ export default Component.extend({
   // == Functions =============================================================
 
   setShift (event) {
-    if (!this.isDestroyed) {
-      this.set('_isShiftDown', event.shiftKey)
-    }
+    run.next(() => {
+      if (!this.isDestroyed) {
```

Comment:
I don't think `this.set` will cause errors if the component is being destroyed but I can update it.